src/main/java/webserver/request/header/RequestHeaderInfos.java

```diff
@@ -0,0 +1,41 @@
+package webserver.request.header;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public class RequestHeaderInfos {
+    private static final int REQUEST_LINE_NEXT_IDX = 1;
+    private static final String HEADER_DELIMITER = ": ";
+    private static final String HOST = "Host";
+    private static final String CONTENT_LENGTH = "Content-Length";
+    private static final String DEFAULT_LENGTH = "0";
+    public static final String DEFAULT_VALUE = "";
+    public static final String COOKIE = "Cookie";
+    private final Map<String, String> infos;
```

Comment:
```suggestion
    public static final String COOKIE = "Cookie";

    private final Map<String, String> infos;
```

상수와 필드 사이의 띄어쓰기가 하나 있으면 좋을 것 같아요 :)

https://google.github.io/styleguide/javaguide.html#s4.6.1-vertical-whitespace